cmds.c

```diff
@@ -3530,6 +3530,12 @@ _c_xfer_file(ch_t * sch,
 	if (ecl || ecr || ec)
 		cr = CMD_ERR_GET;
 
+	if (ec  && strncmp(ec_get_errmsg(ec), "550 File exists", 15) == 0 ||
+	    ecr && strncmp(ec_get_errmsg(ecr),"550 File exists", 15) == 0)
```

Comment:
Nitpick: should be more robust to compare against just the number. According to https://tools.ietf.org/html/rfc959 4.2, the reply is defined to be a 3-digit code followed by a space, but the text afterward can be anything.
```suggestion
	/* file exists */
	if (ec  && strncmp(ec_get_errmsg(ec), "550 ", 4) == 0 ||
	    ecr && strncmp(ec_get_errmsg(ecr),"550 ", 4) == 0)
```

Reply:
I had considered that, then thought not to since 550 is a bit more generic (there can be different reasons than that the file already exists in which can we might need to delete it after all), but I think you're right it's probably still the most reliable check now. I now check just on the 550+space (in case anyone ever would come up with error 5501).
I also am stil considering whether we should include other numbers, or perhaps even revert: there are only a few cases in which we actually want to do the delete.
Ideally we should check beforehand whether the file exists and overwriting is not allowed, but the FEAT string is identical whether overwriting is allowed or not.